sw/device/exts/common/flash_link.ld

```diff
@@ -64,6 +64,19 @@ SECTIONS {
     *(.vectors)
   } > eflash
 
+  /**
+   * For LLVM profiling. This contains a pointer to the runtime initialization
+   * function that is generated by the compiler. See
+   * 'InstrProfiling::emitInitialization()' in 'InstrProfiling.cpp' and
+   * 'getInstrProfInitFuncName()' in 'InstrProf.h'.
+   */
+  .init_array : ALIGN(4) {
+      _init_array_start = .;
+      *(.init_array)
+      *(.init_array.*)
+      _init_array_end = .;
```

Comment:
I would include a `. = ALIGN(4);` here.

Reply:
Thanks for catching this. I think I misunderstood @mcy's comment assuming it was to make sure that the section ends at a word boundary since `_init_array_end` is only used in a comparison. Done.